test/e2e/autoscale_test.go

```diff
@@ -281,7 +281,7 @@ func assertAutoscaleUpToNumPods(ctx *testContext, numPods int32) {
 	defer close(stopChan)
 
 	go func() {
-		if err := generateTraffic(ctx, int(numPods*10), 30*time.Second, stopChan); err != nil {
+		if err := generateTraffic(ctx, int(numPods*10), 60*time.Second, stopChan); err != nil {
```

Comment:
These changes stabilize the autoscaling tests. They have recently been adjusted to continue generating more traffic as soon as the we hit the desired replica count. However that's only been done on "Replicas" so we're at danger of overflowing if the pod takes a while to come up.

Likewise the amount of traffic being sent in (30s) can be juuuuuust about enough to cause us to scale up. After 60s it's guaranteed to (for the default window sizes).